.readthedocs.yml

```
@@ -12,6 +12,7 @@ formats:
 
 sphinx:
```

Shall we declare the version as mentionned in RTD email ?

```suggestion
version: 2
sphinx:
```

---

The version is already at the top of the file.